apis/apps/v1beta3/emqxenterprise_types.go

```diff
@@ -22,8 +22,12 @@ import (
 )
 
 type License struct {
-	Data       []byte `json:"data,omitempty"`
+	// Data represents the byte data of the license content
+	Data []byte `json:"data,omitempty"`
+	// StringData represents the string data of the license content
 	StringData string `json:"stringData,omitempty"`
+	// SecretName represents the license content from the specified secret
```

Comment:
secretName is the name of the secret in the pod's namespace to use. More info: https://kubernetes.io/docs/concepts/storage/volumes#secret